packages/angular-output-target/src/utils.ts

```diff
@@ -86,15 +86,56 @@ export async function readPackageJson(config: Config, rootDir: string) {
 }
 
 /**
- * Creates an import statement for a list of named imports from @angular/core.
+ * Creates an import statement for a list of named imports from a module.
  * @param imports The list of named imports.
+ * @param module The module to import from.
+ *
  * @returns The import statement as a string.
  */
-export const createAngularCoreImportStatement = (imports: string[]) => {
+export const createImportStatement = (imports: string[], module: string) => {
   if (imports.length === 0) {
     return '';
   }
-  return `import { ${imports.join(', ')} } from '@angular/core';`;
+  return `import { ${imports.join(', ')} } from '${module}';`;
+};
+
+/**
+ * Creates the collection of import statements for a component based on the component's events type dependencies.
+ * @param componentTagName The tag name of the component (pascal case).
+ * @param events The events compiler metadata.
+ * @param options The options for generating the import statements (e.g. whether to import from the custom elements directory).
+ * @returns The import statements as an array of strings.
+ */
+export const createComponentEventTypeImports = (
+  componentTagName: string,
+  events: ComponentCompilerEvent[],
+  options: {
+    componentCorePackage: string;
+    includeImportCustomElements?: boolean;
+    customElementsDir?: string;
+  }
+) => {
+  const { componentCorePackage, includeImportCustomElements, customElementsDir } = options;
+  const imports: string[] = [];
+  const namedImports: { [key in string]: boolean } = {};
```

Comment:
No action required - it might be more idiomatic to use a `Record` here
```suggestion
  const namedImports: Record<string, boolean> = {};
```

Reply:
Alternatively, you could probably accomplish the same behavior here with a `Set<string>`